auto_install/install.sh

```diff
@@ -475,6 +475,32 @@ update_repo() {
     echo " done!"
 }
 
+setCustomProto() {
```

Comment:
@StephenKinger @0-kaladin: added the message and some cleanup. Should be good to go